internal/oem/oem.go

```diff
@@ -127,22 +111,6 @@ func init() {
 		name:  "pxe",
 		fetch: noop.FetchConfig,
 	})
-	configs.Register(Config{
-		name:  "rackspace",
-		fetch: noop.FetchConfig,
-	})
-	configs.Register(Config{
-		name:  "rackspace-onmetal",
-		fetch: noop.FetchConfig,
-	})
-	configs.Register(Config{
-		name:  "vagrant",
-		fetch: noop.FetchConfig,
-	})
-	configs.Register(Config{
```

Comment:
Does anything use this?

Reply:
It's a real OEM on CL.  I don't think we should continue it in FCOS, though.

Reply:
Not on this branch!